frameworks/projects/Core/src/main/flex/org/apache/flex/utils/Language.as

```diff
@@ -361,5 +370,56 @@ package org.apache.flex.utils
             return boundMethod;
         };
 
+		/**
+		 * @author lizhi
+		 * @param	arr
+		 * @param	names
+		 * @param	opt
+		 */
+		public static function sortOn(arr:Array,names:Object,opt:int=0):void{
```

Comment:
Sort options can also be an array if names is an array http://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/Array.html#sortOn()